helm/hpcc/values.schema.json

```diff
@@ -2946,6 +2967,131 @@
+            },
+            "scaleDown": {
+              "type": "object",
+              "$ref": "#/definitions/hpaScalingRules"
+            }
+          }
+        },
+        "metrics": {
+          "type": "array",
+          "description": "Specifies variables used to calculate the desired replica count. The desired replica count is calculated multiplying the ratio between the target value and the current value by the current number of pods.",
+          "items": { "$ref": "#/definitions/hpaMetric" }
+        }
+      },
+      "required": [ "maxReplicas", "metrics" ]
+    },
+    "hpaScalingRules": {
+      "type": "object",
+      "properties": {
+        "policies" : {
+          "type": "array",
+          "items": {
+            "type": "object",
+            "properties": {
+              "type": {
+                "type": "string",
+                "description": "Type is used to specify the scaling policy"
+              },
+              "value": {
+                "type": "integer",
+                "description": "Value contains the amount of change which is permitted by the policy. It must be greater than zero"
+              },
+              "periodSeconds": {
+                "type": "integer",
+                "description": "Specifies the window of time for which the policy should hold true. PeriodSeconds must be greater than zero and less than or equal to 1800 (30 min)."
+              }
+            }
+          }
+        },
+        "selectPolicy": {
+          "type": "string",
+          "description": "selectPolicy is used to specify which policy should be used. If not set, the default value Max is used."
+        },
+        "stabilizationWindowSeconds": {
+          "type": "integer",
+          "description": "Seconds for which past recommendations should be considered while scaling up or scaling down. StabilizationWindowSeconds must be greater than or equal to zero and less than or equal to 3600 (one hour). If not set, use the default values: - For scale up: 0 (i.e. no stabilization is done). - For scale down: 300 (i.e. the stabilization window is 300 seconds long)."
+        }
+      }
+    },
+    "hpaMetric" : {
+      "type": "object",
+      "description": "Target metric for Horizontal Pod Autoscaler",
+      "properties": {
+        "type": {
+          "type": "string",
+          "description" : "The metric type to determine workload scaling",
+          "enum": [ "Resource","Pods", "External", "Object"]
+        },
+        "name": {
+          "description": "The metric name",
+          "type": "string"
+        },
+        "selector": {
+          "type": "object",
+          "description": "selector is the string-encoded form of a standard kubernetes label selector for the given metric When set, it is passed as an additional parameter to the metrics server for more specific metrics scoping. When unset, just the metricName will be used to gather metrics.",
```

Comment:
re:  https://github.com/hpcc-systems/HPCC-Platform/pull/17132#discussion_r1164378310

not a big deal I think, but were you going to normalize the capitalization in the descriptions that refer to the properties, e.g. this one "selector" is referred to in the correct case, but others e.g. "Value" and "Type" (lines 3013, 3017) are not. IMO, it is prob. best if the description describes without referring to the property name to avoid this issue.